Purchases/Purchasing/StoreKitAbstractions/SK1StoreTransaction.swift

```diff
@@ -71,4 +67,19 @@ extension SKPaymentTransaction {
         return identifier
     }
 
+    fileprivate var quantity: Int {
+        return self.maybePayment?.quantity ?? 1
+    }
+
+    /// Considering issue https://github.com/RevenueCat/purchases-ios/issues/279, sometimes `payment`
+    /// and `productIdentifier` can be `nil`, in this case, they must be treated as nullable.
+    /// Due of that an optional reference is created so that the compiler would allow us to check for nullability.
```

Comment:
`Due of that` -> `Due to that,`
Typo in the old comment.

Reply:
Thanks fixed.